packages/eslint-rules/lib/rules/no-custom-color.js

```diff
@@ -20,6 +20,7 @@ const alwaysValidClasses = [
   'bg-no-repeat',
   'bg-center',
   'bg-contain',
+  'text-opacity-64', // refactor in a separate PR to something that would filter all text-opacity-*
```

Comment:
Out of scope, but looking at this, should we add a ticket to rename the rule to `no-custom-classes.js`? 🤔

Reply:
Agreed. Will write one for this.